tests/k8s/tests/00-gsg-test.sh

```diff
@@ -25,10 +25,10 @@ K8SDIR="${dir}/../../../examples/kubernetes"
 GSGDIR="${dir}/deployments/gsg"
 
 function cleanup {
-	kubectl delete -f "${MINIKUBE}/l3_l4_l7_policy.yaml" 2> /dev/null || true
-	kubectl delete -f "${MINIKUBE}/l3_l4_policy_deprecated.yaml" 2> /dev/null || true
-	kubectl delete -f "${MINIKUBE}/l3_l4_policy.yaml" 2> /dev/null || true
-	kubectl delete -f "${GSGDIR}/demo.yaml" 2> /dev/null || true
+  kubectl delete -f "${MINIKUBE}/l3_l4_l7_policy.yaml" 2> /dev/null || true
```

Comment:
This could be made on a different PR, it's not easy to review 30 commits on a single PR